website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -453,11 +449,9 @@ $ make undeploy
 
 ## Further steps
 
-The following guides build off the operator created in this example, adding advanced features:
-
-- [Create a validating or mutating Admission Webhook][create_a_webhook]
+Now, check how to integrate your project with [OLM][olm-integration] to know how to distribute your project and, for example, publish your projects in the https://operatorhub.io/. 
```

Comment:
The OLM integration should be listed as a first-class deployment option, not an afterthought.

Reply:
Are you saying that we should suggest the user check ho to integrate the OLM before teach ho to make deploy the project running? IMO first I need to check the project running with `make deploy` then we can teach how to integrate it with OLM.